test/lib/ServiceLocator.js

```diff
@@ -147,4 +147,20 @@ describe('ServiceLocator', function() {
 			});
 		});
 	});
+
+	describe('#has', function() {
+		it('should return true', function() {
```

Comment:
should be a reason like `should return true when it has the registered type by the name`